packages/orchestration/src/exos/portfolio-holder-kit.js

```diff
@@ -0,0 +1,200 @@
+import { M, mustMatch } from '@endo/patterns';
+import { E } from '@endo/far';
+import { VowShape } from '@agoric/vow';
+import { Fail } from '@endo/errors';
+import {
+  TopicsRecordShape,
+  PublicTopicShape,
+} from '@agoric/zoe/src/contractSupport/topics.js';
+import { makeScalarBigMapStore } from '@agoric/vat-data';
+
+const { fromEntries } = Object;
+
+/**
+ * @import {MapStore} from '@agoric/store';
+ * @import {VowTools} from '@agoric/vow';
+ * @import {ResolvedPublicTopic, ResolvedTopicsRecord} from '@agoric/zoe/src/contractSupport/topics.js';
+ * @import {Zone} from '@agoric/zone';
+ * @import {CopyMap} from '@endo/patterns';
+ * @import {OrchestrationAccount} from '@agoric/orchestration';
+ * @import {ResolvedContinuingOfferResult} from '../utils/zoe-tools.js';
+ */
+
+/**
+ * @typedef {{
+ *   accounts: MapStore<string, OrchestrationAccount<any>>;
+ *   publicTopics: MapStore<string, ResolvedPublicTopic<unknown>>;
+ * }} PortfolioHolderState
+ */
+
+const ChainNameM = M.string();
```

Comment:
like all the other shapes:
```suggestion
const ChainNameShape = M.string();
```